src/Mod/CAM/Path/Tool/shape/ui/shapeselector.py

```diff
@@ -67,10 +67,16 @@ def update_shapes(self):
         # Retrieve each shape asset
         builtin = cam_assets.fetch(asset_type="toolbitshape", store="builtin")
         builtin = {c.id: c for c in builtin}
+
+        turning = cam_assets.fetch(asset_type="toolbitshape", store="builtin")
```

Comment:
@knipknap I have added all tools to a new Turning page. What would be the best way to filter tools for each page?

Reply:
I thought about that but it seemed a bit hacky 🧐. Maybe later we could add some additional properties to tools to allowing defining what that can be used for?

Reply:
That kind of use was actually the intent of introducing the mixins. What would properties offer that the mixins don't?